provisioning/ubuntu-provision.sh

```diff
@@ -386,6 +386,7 @@ function install_datadog() {
 
   apt-get update
   apt-get install --yes --no-install-recommends datadog-agent datadog-signing-keys
+  systemctl disable datadog-agent
```

Comment:
```suggestion
  # Disabling service at startupt because <....>
  systemctl disable datadog-agent
```

Do you mind adding a comment to explain "why" ? Otherwise LGTM